lib/flipper/api/v1/actions/boolean_gate.rb

```diff
@@ -0,0 +1,21 @@
+require 'flipper/api/action'
+
+module Flipper
+  module Api
+    module V1
+      module Actions
+        class BooleanGate < Api::Action
+          route %r{api/v1/features/[^/]*/(enable|disable)/?\Z}
+          
+          def put
+            feature_name = Rack::Utils.unescape(request.path.split("/")[-2])
+            feature = flipper[feature_name.to_sym]
+            action = Rack::Utils.unescape(request.path.split("/").last)
```

Comment:
> Could see some idea of this being moved into Api::Action at some point since just about every action uses some form of splitting on "/" and accessing an index

Yeah that would be totally fine by me. Good call.